### PR TITLE
Cloud release 24.2.0 - 28 August 2024

### PR DESCRIPTION
**Pull request by justinegeffen**
Release notes here: https://github.com/seqeralabs/platform/releases/tag/v24.2.0_cycle23_1705fd6
Selected changes added to this PR.

### CODE REVIEW

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 1]
```diff
@@ -5,6 +5,12 @@ date: "4 July 2024"
 tags: [changelog]
 ---
 
+### v24.2.0_cycle23 Cloud - 28 August 2024
+
+Fix: [Data Explorer] enable task workdir navigation using Data Explorer in personal workspaces by adding routing
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:10]:
```suggestion
Fix: [Data Explorer] enable task working directory navigation using Data Explorer in personal workspaces by adding routing
```

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 2]
```diff
@@ -5,6 +5,12 @@ date: "4 July 2024"
...
+Fix: Remove -/+ increment buttons from numeric input components in Platform, including Compute Environment and Launch Form interfaces
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:11]:
```suggestion
Fix: Remove -/+ increment buttons from numeric input components in Platform, including compute environment and launch form interfaces
```

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 3]
```diff
@@ -5,6 +5,12 @@ date: "4 July 2024"
...
+Fix: Hide compute environment variables section actions when component is disabled (previously indicated that env vars could be edited in this state)
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:12]:
```suggestion
Fix: Hide compute environment variables section actions when component is disabled (previously indicated that environment variables could be edited in this state)
```

**Review by llewellyn-sl — COMMENTED** [review]
Some minor comments. LGTM otherwise

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 4]
```diff
@@ -5,6 +5,12 @@ date: "4 July 2024"
 tags: [changelog]
 ---
 
+### v24.2.0_cycle23 Cloud - 28 August 2024
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:8]: We'll need to update the date here to the actual release date before we publish.

**Review by llewellyn-sl — APPROVED** [review]
Approved, contingent upon updating the release date when we know what it is.

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx (hunk 4 above)
Comment (justinegeffen) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:8]:
```suggestion
### v24.2.0_cycle23 Cloud - 2 September 2024
```

**Pull request merged**